### PR TITLE
OpenCV.js challenges 

### LINKED ISSUE #36384: OpenCV.js tutorial at freeCodeCamp.org
Greetings!
We are Summer Intel Interns.
Our team would like to collaborate with freeCodeCamp.org project.

We want to create beginner's course about OpenCV.js. The course will include topics like: installing, resize image and image processing, camera management and others by means of short theory sentences, interactive examples, tasks for self test.

Based on this, better way to create this course is challenges: https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md

But we can consider to create that course in other ways you accept:
YouTube: https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ
Articles: https://terms-of-service.freecodecamp.org

We want to ask you several questions about content requirements
 on freeCodeCamp.org platform:
1) Are you considering that kind of contributions to your platform?
2) In which category do you want to see our course? (https://learn.freecodecamp.org)
3) What expectations and proposals do you have about that project?

We will be glad to hear your ideas and offerings.

Sincerely, @ilbalance, @SimpleVlad.

/cc @dkurt.

### PR DESCRIPTION
**Pull request by SimpleVlad**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #36384

@RandellDawson, @thecodingaviator, @raisedadead, @scissorsneedfoodtoo, @jonathan-grah, we are pleasant to present initial version of Image processing and Computer vision challenges based on OpenCV.js. Please review. Looking forward for your feedback.  

DONE:
Tracklist of samples: 
1)convert color image to grayscale
2)smoothing image
3)modify contrast and brightness
4)resize an image
5)rotate an image
6)blend images
7)find contours
8)draw on image
9)image recovery

TODO:
- [ ] For now we put our sample in freeCodeCamp/curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/. Which location is more appropriate for this? After we decide certain location, we gonna add suffixes .english.md  .
- [ ] We are going to provide small version of OpenCV.js (public one takes about 8MB). In which place we can put it?
- [ ] We also can add samples with camera management (in example, show frames on the screen). However we doubt that not all the users have USB cameras. What is better, to mark it as optional or not to add camera samples at all?

/cc @dkurt, @ilbalance, @SimpleVlad

### CODE REVIEW

**Comment by thecodingaviator** [comment]
I’m not sure if we’re actually ready to go through with this (cc: @QuincyLarson ) but as far as the positioning of the challenge is concerned, it should go somewhere in the front end libraries section as I guess that’s what this is

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo According to [Quincy's suggestions on issue #36384](https://github.com/freeCodeCamp/freeCodeCamp/issues/36384#issuecomment-510891166) (see below), this was not supposed to be added as part of the current curriculum.  I took it to mean it would be part of the new curriculum but as a supplement.  I am fairly sure it should not be in the `Basic HTML and HTML5` section.

* Create an in-depth text-based tutorial
* Turn that tutorial into a video
* Turn that into an interactive curriculum

The PR created does not seem to address any of Quincy's suggestions.

**Comment by dkurt** [comment]
@RandellDawson, May I ask you to share some example with new curriculum so we can move the challenges to it? This is just initial implementation and we can modify it in any format you expect to see. Thanks!

**Comment by jonathan-grah** [comment]
Yeah, I don't think we are adding any additional challenges atm due to our refactor to a more project-based approach (https://www.freecodecamp.org/forum/t/help-us-build-version-7-0-of-the-freecodecamp-curriculum/263546).

**Comment by dkurt** [comment]
@jonathan-grah, Can we already touch this approach? Maybe there is some branch which we can refer?

**Comment by raisedadead** [comment]
Hi @dkurt @SimpleVlad thanks for the pull-request, you are more than welcome to work on your own forks as of now and keep us updated on the progress on the original issue thread (now transferred to the discussion repo).

When we are ready to accept PRs, we would let you know on the original issue thread and request you to make PRs.

As of now, you can just post your challenge ideas on the issue thread.

**TL;DR:** This repository is not ready for the new curriculum.

Please reachout to the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors) for further help, as of now I am going to close this PR because we are not going to land these changes as it is.

**Comment by scissorsneedfoodtoo** [comment]
@SimpleVlad, I left a comment here: https://github.com/freeCodeCamp/CurriculumExpansion/issues/209#issuecomment-525109551
